less/navigationBar.less

```diff
@@ -29,6 +29,393 @@
+  }
+
+  button.captionButton {
+    outline: none;
+    vertical-align: top;
+  }
+
+  &.fullscreen {
+    .win7 {
+      margin-top: 1px;
+    }
+  }
+
+  .win7 {
+    margin-right: 6px;
+
+    button.captionButton {
+      outline: 0;
+      height: 18px;
+      color: #b1acac;
+      border: 1px solid #838383;
+      border-top: 0;
+      display: inline-block;
+      background-color: #e0e0e0;
+      box-shadow: inset 1px 1px rgba(255, 255, 255, 0.9);
+      width: 25px;
+
+      &.minimize {
+        width: 28px;
+        border-right: 0px;
+        &:hover {
+          background-color: #f5f5f5;
+        }
+        &:active {
+          background-color: #cacacb;
+        }
+        border-radius: 0 0 0 4px;
+
+        .widget {
+          width: 10px;
+          height: 3px;
+          border: 1px solid #838383;
+          background: #fefefe;
+          display: inline-block;
+          border-radius: 1px;
+        }
+      }
+
+      &.maximize {
+        border-right: 0px;
+        width: 27px;
+        &:hover {
+          background-color: #f5f5f5;
+          .widget {
+            .widget2 {
+              background-color: #f5f5f5;
+            }
+          }
+        }
+        &:active {
+          background-color: #cacacb;
+          .widget {
+            .widget2 {
+              background-color: #cacacb;
+            }
+          }
+        }
+        &.fullscreen {
+          &:hover {
+            background-color: #e5e5e5;
+          }
+          &:active {
+            background-color: #cacacb;
+          }
+          .widget {
+            .widget1 {
+              width: 8px;
+              top: 2px;
+              left: 10px;
+            }
+            .widget2 {
+              width: 8px;
+              height: 8px;
+              top: -5px;
+              left: 6px;
+              background: #fefefe;
+              border-radius: 1px;
+            }
+            .widget3 {
+              display: inline-block;
+              width: 2px;
+              height: 2px;
+              border: 1px solid #838383;
+              background: #fefefe;
+              position: relative;
+              top: -20px;
+              left: -2px;
+            }
+          }
+        }
+
+        .widget {
+          .widget1 {
+            width: 10px;
+            height: 8px;
+            border: 1px solid #838383;
+            background: #fefefe;
+            position: relative;
+            top: 2px;
+            left: 7px;
+            border-radius: 1px;
+          }
+          .widget2 {
+            width: 4px;
+            height: 2px;
+            border: 1px solid #838383;
+            background-color: #e0e0e0;
+            position: relative;
+            top: -5px;
+            left: 10px;
+            border-radius: 0;
+          }
+          .widget3 { display: none; }
+          .widget4 { display: none; }
+          .widget5 { display: none; }
+        }
+      }
+
+      &.close {
```

Comment:
If you mouse over the close button and slowly move your mouse up past the top of the window, the red stays solid.